lib/origami-build-tools-cli.js

```diff
@@ -96,6 +96,8 @@ if (tasks.isValid(argument)) {
 				taskResult
 					.on('error', reportTaskError)
 					.on('end', reportFinished);
+				// Make sure it reaches 'end'
+				taskResult.resume();
```

Comment:
I have already pushed a change to that a while ago. I'm trying to find out why the build gets stuck